src/main/java/io/appium/java_client/flutter/FlutterDriver.java

```diff
@@ -0,0 +1,23 @@
+package io.appium.java_client.flutter;
+
+import org.openqa.selenium.WebDriver;
+
+/**
+ * The {@code FlutterDriver} interface represents a driver that controls interactions with
+ * Flutter applications, extending WebDriver and providing additional capabilities for
+ * interacting with Flutter-specific elements and behaviors.
+ *
+ * <p> This interface serves as a common entity for drivers that support Flutter applications
+ * on different platforms, such as Android and iOS. </p>
+ *
+ * @see WebDriver
+ * @see SupportsGestureOnFlutterElements
+ * @see SupportsScrollingOfFlutterElements
+ * @see SupportsWaitingForFlutterElements
+ */
+public interface FlutterDriver extends
```

Comment:
Lets maybe make this interface name more specific, like SupportsFlutterIntegrationScripts

Reply:
can we rename it as `FlutterIntegrationTestDriver` to be more specific similar to how we have `WebDriver`?

Reply:
I mean that both AndroidDriver and IOSDriver have this interface, so it rather looks redundant to me to also have it here. interfaces should be kept atomic and minimal. It is always better to have two interfaces declaring two separate things than one big one combining both